src/components/SUE/report/SueReportSend.tsx

```diff
@@ -58,15 +67,26 @@ export const SueReportSend = ({
     const formData = {
       dataType: 'json',
       dataSource: 'form',
-      content: JSON.stringify(createAppUserContentNewData)
+      content: JSON.stringify({
+        action: 'notify_and_destroy',
+        appVersion: appJson.expo.version,
+        created_At: moment().format('YYYY-MM-DD HH:mm:ss'),
+        message: createAppUserContentNewData.message,
+        rating: createAppUserContentNewData.ratingCount
+      })
```

Comment:
what about `...createAppUserContentNewData, other data`? this also needs to name `ratingCount` to `rating`, right!? it seems strange to have a different name in here, we can use the name from the api

Reply:
Yes, that's right. I will update. 👍